src/main/java/org/opentripplanner/routing/impl/SoundTransitSounderFares.java

```diff
@@ -0,0 +1,261 @@
+package org.opentripplanner.routing.impl;
+
+import com.google.common.collect.ImmutableMap;
+import org.opentripplanner.routing.core.Fare;
+
+import java.util.Map;
+
+public class SoundTransitSounderFares {
+    public static void populateSounderFares(Map<String, Map<Fare.FareType, Float>> linkLightRailFares) {
+        linkLightRailFares.put(
```

Comment:
For this and the SoundTransitLinkFares class, a matrix may be a better data structure to reason about and organize these fares.

Reply:
Hm, you're probably right. I was just following the existing structure and wrote a Python script to generate the Java code that you see in the populate methods from the HTML of the agency's fares page. It would probably more sense to keep it in a CSV which is processed by OTP. 

The thing is, I'm not sure if it's really worth putting the time into creating a new data structure for this, though, with GTFS Fares V2 theoretically coming, which should supersede the need for these hard coded values.